radio/src/targets/common/arm/stm32/usbd_cdc.cpp

```diff
@@ -223,6 +223,13 @@ static uint16_t VCP_DataRx (uint8_t* Buf, uint32_t Len)
     cliRxFifo.push(Buf[i]);
   }
 #endif
+#if defined(LUA) && !defined(CLI)
+  // copy data to the LUA FIFO
+  for (uint32_t i = 0; i < Len; i++)
```

Comment:
code style

Reply:
Fixed the brackets, was that what you meant?